packages/feedback-react/src/questions/ContactQuestion.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { ReactNode, useState } from "react";
+import { PrimaryButton, TertiaryButton } from "@fremtind/jkl-button-react";
+import { TextInput } from "@fremtind/jkl-text-input-react";
+import { useFeedbackContext } from "../feedbackContext";
+
+interface Props {
+    label?: string;
+    sendButtonLabel?: string;
+    withPhone?: boolean;
+    children?: ReactNode;
+    onSubmit: (values: { email: string; phone?: string }) => void;
+}
+
+export const ContactQuestion = ({
+    label = "Kan vi kontakte deg for flere innspill?",
+    sendButtonLabel = "Sett meg på lista!",
+    withPhone = false,
+    onSubmit,
+    children,
+}: Props) => {
+    const [noThanks, setNoThanks] = useState(false);
+    const [email, setEmail] = useState("");
+    const [phone, setPhone] = useState("");
+    const { contactSubmitted, setContactSubmitted } = useFeedbackContext();
+
+    const ChildrenWrapper = typeof children === "string" ? "p" : "div";
+
+    const handleSubmit: React.FormEventHandler = (e) => {
+        e.preventDefault();
+        onSubmit({ email, phone: withPhone ? phone : undefined });
+        setContactSubmitted(true);
+    };
+
+    if (noThanks || contactSubmitted) {
+        return null;
+    }
+
+    return (
+        <form className="jkl-layout-spacing--medium-top" onSubmit={handleSubmit}>
+            <p className="jkl-heading-4 jkl-component-spacing--small-bottom">{label}</p>
+            {children && <ChildrenWrapper className="jkl-body">{children}</ChildrenWrapper>}
+
+            <TextInput
+                className="jkl-layout-spacing--small-top"
+                variant="small"
+                label="E-post"
+                autoComplete="email"
+                name="email"
+                value={email}
+                onChange={(e) => setEmail(e.target.value)}
```

Comment:
Skulle det vært et minimum av skjemavalidering for epost og telefonnummer? Typ `^.+@.+\..+$` for epost og `^\d{8}$` for telefon?

Reply:
Ja, er nok greit! Det sto på listen min, men ble visst litt ivrig med omskriving 😅

Reply:
Eventuelt om man også kunne lagt inn valideringen selv. Jeg har hørt at de nå benytter en API-tjeneste for å validere e-post i FIP for å få høyere datakvalitet.

Reply:
Samme for telefon, men å ha minimumvalidering for telefon er kanskje litt mer tricky

Reply:
I flere av prosjektene, blant annet kfp-skade og kfp-liv har vi brukt react-form-hooks for håndtering og validering av forms, det burde være mulig å integrere med det også kanskje?

Reply:
> Jeg har hørt at de nå benytter en API-tjeneste for å validere e-post i FIP for å få høyere datakvalitet.

Jeg tror ikke denne komponenten er stedet for så innfløkt validering. Her ønsker vi så lite friksjon som mulig for brukeren, slik at vi faktisk får inn informasjonen. Jeg brukte valideringsreglene fra `jkl-validators-util` 😊 Vi forsøkte også `react-hook-form`, men det mer enn tredoblet størrelsen på komponenten, og det er tricky å ha som peer dependency pga API-forandingene mellom v6 og v7, siden begge brukes i klientene våre